cb_hr_views/models/res_partner.py

```diff
@@ -45,7 +45,7 @@ def toggle_active_modified(self):
             )
             if user_ids:
                 user_ids.write({"active": active})
-            if not active:
+            if not active or (not user_ids and active):
```

Comment:
Why dis condition, you need to change the active of the partner, isn't it?

Reply:
Maybe , it could be easier to do 
```
if not user_ids:
    record.toggle_active()
    continue
```

Reply:
About the toggle_active, it is not redefined for res.partner, so it should be the same to use ir or not. Maybe other modules override or inherit toggle active, but i dont know if that makes it safer to use or not in this specific case.

About the condition, unarchiving a user also unarchives the partner, but archiving the user does not archive the partner so you have to enter if not user_ids, of course, but also when you are archiving a partner, even if it has users, because as I said before, the `user_ids.write({"active": active})` wont propagate to res.partner in that case.

Anyways now I realize that the condition can be expressed as `if not user_ids or not active` and it is exactly the same.